ui/packages/components/.storybook/preview.tsx

```diff
@@ -1,14 +1,20 @@
-import { AppRoot } from '@inngest/components/AppRoot';
 import { withThemeByClassName } from '@storybook/addon-themes';
 import type { Preview, ReactRenderer } from '@storybook/react';
 
+import { interTight, robotoMono } from '../src/AppRoot/fonts';
+import '../src/AppRoot/globals.css';
+
 const preview: Preview = {
   decorators: [
     (Story) => {
       return (
-        <AppRoot>
+        <div
```

Comment:
I detailed it in the commit message 🙂: https://github.com/inngest/inngest/pull/817/commits/aaa0d9c4d41f7ddd77c612e018cfc57b79afb3f6

Pretty much React is now stricter and doesn't allow multiple components that contain an `html`, `head, or `body` tag to be mounted at the same time. When this occurs, React will replace the previously mounted component with the new one. This breaks our Storybook because `AppRoot`, which contains those tags, is wrapping every story and we display multiple stories in a same Storybook page.

Reply:
Gotcha, makes sense